### PR TITLE
Fabo/changelog tool

### PR DESCRIPTION
**Pull request by faboweb**
Experimenting with changelog conflict management (inspired by Alessio):

Run `yarn changelog` to add changes (will create a file for your branch)
Test final changelog: `node tasks/createReleasePR.js` (attention: creates a commit and a `bot` remote)

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `PENDING.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: changes/fabo_changelog-tool [hunk 1]
```diff
@@ -0,0 +1 @@
+[Added] Added a tool to handle changelog conflicts @faboweb
```
Comment (fedekunze) [inline changes/fabo_changelog-tool]: shouldn't his go to `added/...`

Comment (fedekunze) [reply changes/fabo_changelog-tool]: also should we rename the file ?

Comment (faboweb) [reply changes/fabo_changelog-tool]: I chose to go with `changes/{branch name}` so you are only responsible for the changes of your branch which shouldn't create conflicts. Going with `changes/added/....` we would create a lot more lookups and I think that would be more complicated to handle the folder structures.

Comment (fedekunze) [reply changes/fabo_changelog-tool]: ohh ok, I wasn't aware of that naming standard. Yeah, I agree with your point, just let's make sure we document that on `CONTRIBUTING.md`

File: tasks/createReleasePR.js [hunk 2]
```diff
@@ -3,8 +3,11 @@
 const { cli, shell } = require(`@nodeguy/cli`)
 const fs = require(`fs`)
 const { join } = require(`path`)
+const _ = require(`lodash`)
```
Comment (sabau) [inline tasks/createReleasePR.js]: Even if it's not in the bundle is nice to import with { groupBy } :)

**Comment by fedekunze** [comment]
this is not documented at all, we need to add the docs ASAP

**Comment by fedekunze** [comment]
command doesn't work too
```
MacBook-Pro:lunie federico$ yarn changelog
yarn run v1.15.2
$ node tasks/changelog.js
? What type of change do you want to add to the changelog? 
  Addition of feature 
❯ Change of existing behavior 
  Fix for a bug 
  Security improvement 
  Deprecation of unused code/feature (node:29009) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open '../changes/fedekunze_2323-anchor'
? What type of change do you want to add to the changelog? 
  Addition of feature 
❯ Change of existing behavior 
  Fix for a bug 
  Security improvement 
  Deprecation of unused code/feature 
```

File: tasks/changelog.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        validate: function(value) {
+          if (value) return true
+
+          return `You need to specify the change.`
+        }
+      },
+      {
+        type: `input`,
+        name: `issue`,
+        message: `(optional) Which GitHub issue is discussing this change?`
```
Comment (fedekunze) [inline tasks/changelog.js]: this should be mandatory

**Comment by sabau** [comment]
So I'll restore the branch and you open a revert one 👍 ?

**Pull request merged**